src/components/UpcomingShiftCard/UpcomingShiftCard.css

```
@@ -0,0 +1,26 @@
+.card{
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 1px 4px;
+  width: 400px;
+  border: none;
+}
+.card-body{
+  padding: 15px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.card-title{
+  margin: 0;
+}
+.arrow {
+  border: solid #6666FF;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 5px;
+  transform: rotate(-45deg);
+  -webkit-transform: rotate(-45deg);
+}
+.card-link{
+  color: #6666FF;
+  font-weight: lighter;
+}
```

Don't think you need most of this css because you're using Bootstrap now
```suggestion
.arrow {
  border: solid #6666FF;
  border-width: 0 3px 3px 0;
  display: inline-block;
  padding: 5px;
  transform: rotate(-45deg);
  -webkit-transform: rotate(-45deg);
}
```